### ISSUE TITLE
carriage returns in BTA tsv export causing misalignment of data when importing into speadsheets

### ISSUE THREAD

**Issue by jhpoelen** [jhpoelen/bat-taxonomic-alignment#9]
to reproduce:

download a BTA version using

```
curl "https://linker.bio/hash://sha256/981b8f9ece76eb4418fe82e8dfa077165943fe1d63103fa4a25f21a2d7881e75"\
 > bta.tsv
```

open the bta.tsv into a spreadsheet program like LibreOffice Calc. 

Expected to see well-aligned data.

Actual notice apparent truncated lines causing incomplete rows (see screenshot).

Root cause appears to be carriage returns embedded in the tsv file. In `vi` these carriage returns are encoded as `^M` .

```
cat bta.tsv\
 | grep -n "bunkeri"\
  | head -n1 > aline.tsv
```

with 

```cat aline.tsv | wc -l``` yielding "1", indicated that only a single new line exists (`\n`). However, when opening the aline.tsv in LibreOffice Calc, multiple lines are observed, apparently caused by occurrences of carriage returns (`\r` or `^M`). 

![image](https://user-images.githubusercontent.com/1084872/234646130-02d56c77-508c-44eb-9be7-4d78222c7eb2.png)

![image](https://user-images.githubusercontent.com/1084872/234646316-08fe454e-8527-4864-80f3-9148dfabfa15.png)

@n8upham

**Comment by jhpoelen** [comment]
In their original, the carriage lines are visible also in the google sheets editors - 

![image](https://user-images.githubusercontent.com/1084872/234646834-3c717347-1f73-4b5d-a3b1-64ff96315a82.png)

**Comment by jhpoelen** [comment]
See related issue https://techcommunity.microsoft.com/t5/excel/excel-problem-with-importing-csv-file-with-carriage-return-in-a/m-p/1404378 .

**Comment by jhpoelen** [comment]
After replacing the carriage returns with newlines using:

```
cat bta.tsv\
 | tr '\r' ' '\
 > bta-no-carriage-return.tsv
```

the issue no longer exists. 

![image](https://user-images.githubusercontent.com/1084872/234648713-89b1caee-c576-4701-a4db-c13b04258bd1.png)

**Comment by jhpoelen** [comment]
@n8upham I just released a new version of BTA@306cd3c9 via https://jhpoelen.nl/bat-taxonomic-alignment/ 

and using this new version, I was able to load the bta tsv into spreadsheet ok using

```
curl "https://linker.bio/hash://sha256/306cd3c999895af317044442e86fac33769ce178f6670b050683ca0ea80e6c67.tsv" > bta.tsv
```

![image](https://user-images.githubusercontent.com/1084872/234666995-5e1a7eb7-4864-430a-9d66-e53877b59363.png)

**Comment by n8upham** [comment]
Awesome, thanks for letting me know man!
—n
